README.md

```diff
@@ -190,6 +191,11 @@ $ perceval github elastic logstash --from-date '2016-01-01'
 $ perceval gitlab elastic logstash --from-date '2016-01-01'
 ```
 
+### Groups.io
+```
+$ perceval groupsio 'updates' -t 'abcdefghijk' --from-date '2016-01-01'
```

Comment:
The help output says `groups.io` is the way to call the backend but here you're using `groupsio`.